docs/superpowers/plans/2026-07-02-with-auth-wrapper.md

```
@@ -0,0 +1,1339 @@
+---
+
+### Task 4: Update `docs/progress.md`
+
+**Files:**
+- Modify: `docs/progress.md`
+
+- [ ] **Step 1: Add entry to `docs/progress.md`**
+
+Under the "Infrastructure / Auth" section (or the nearest equivalent section), add:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)